frontend/src/components/regions/DueTimeFormModal/Presentation.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import {
+  TextField,
+  Button,
+  Grid,
+  Paper,
+} from "@material-ui/core";
+import Modal from '../../widgets/Modal';
+
+const useStyles = makeStyles({
+
+    dueTimeButton: {
+        marginTop: "0.3rem",
+        marginRight: "0.3rem",
+    },
+
+    paper: {
+      padding: "1rem"
+    }
+});
+
+const DueTimeFormModal = ({ card, closeModal }) => {
+
+  const {dueTime} = "";
```

Comment:
What is this line for?

If the card has a due time that was already set (eg the card is due on Tuesday next week) then we want to pre-populate the form so that it refers to Tuesday next week. We dont want it to always be vblank.

Also, I dont think this code does what you think it does:

```
>   const {dueTime} = "";
> console.log(dueTime)
undefined
```

Your comment is still not entirely clear. Why are you choosing to slice at 16? Why not 133? 
Good code makes its intentions clear. Please make sure that the intention of this line is clear.

Eg: # if the dueTime has a value, then we need to strip out the seconds.